.github/scripts/install_deps.sh

```diff
@@ -6,9 +6,8 @@ set -ev
 ADD_PKG_LIST=""
 
 if [ "${GITHUB_JOB}" = "clang_tidy" ]; then
-  ADD_PKG_LIST="
-    clang-tidy  \
-    python-yaml"
+  pip install pyyaml
```

Comment:
if you look at lone 12 of the log of the dependencies install task we might not need this.